source/braille.py

```diff
@@ -792,7 +792,7 @@ def getControlFieldBraille(info, field, ancestors, reportStart, formatConfig):
 		return (_("%s end") %
 			getBrailleTextForProperties(role=role))
 
-def getFormatFieldBraille(field, isAtStart, formatConfig):
+def getFormatFieldBraille(field, isAtStart, formatConfig, fieldCache=None):

```

Comment:
isAtStart: True if this FormatField precedes any text in the line/paragraph. This is useful to restrict display of information which should only appear at the start of the line/paragraph; e.g. the line number or line prefix (list bullet/number).